pyproject.toml

```diff
@@ -57,9 +57,19 @@ interfaces-vision = [
     "pillow>=6.2.1,<11.0"
 ]
 
+interfaces-ts = [
+    "pandas>=1.4.3,<2",
+]
+
+interfaces-ts-pyspark = [
+    "pandas>=1.4.3,<2",
```

Comment:
```suggestion
interfaces-ts-pyspark = [
    "caikit[interfaces-ts]",
```

Reply:
This will avoid needing to maintain the same dependency range twice for `pandas`